apps/modernization-ui/src/apps/page-builder/components/SaveTemplate/SaveTemplate.spec.tsx

```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { SaveTemplates } from './SaveTemplate';
+
+describe('When page loads', () => {
```

Comment:
Could you change this to describe the component so it is easier to read the tests. Like "describe("SaveTemplate component", () ...